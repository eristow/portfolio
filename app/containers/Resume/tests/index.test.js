/**
 *
 * Tests for Resume
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from 'react-testing-library';
// import 'jest-dom/extend-expect'; // add some helpful assertions

import { Resume } from '../index';

describe('<Resume />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<Resume dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Resume />);
    expect(firstChild).toMatchSnapshot();
  });
});
