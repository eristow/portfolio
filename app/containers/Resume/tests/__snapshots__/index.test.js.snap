// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Resume /> Should render and match the snapshot 1`] = `
<div>
  <h1>
    My Resume
  </h1>
  <a
    href="https://drive.google.com/file/d/1wZxKA1I1gF5xgExKVLuoM8_FpWyKPtHy/view?usp=sharing"
  >
    Resume
  </a>
</div>
`;
